apps/jupyter/templates/services/app/dc.yml.j2

```diff
@@ -0,0 +1,54 @@
+apiVersion: v1
+kind: DeploymentConfig
+metadata:
+  labels:
+    app: jupyter
+    service: app
+    version: "{{ jupyter_image_tag }}"
+  name: jupyter
+  namespace: "{{ project_name }}"
+spec:
+  replicas: {{ jupyter_app_replicas }}
+  template:
+    metadata:
+      labels:
+        app: jupyter
+        service: app
+        version: "{{ jupyter_image_tag }}"
+        deploymentconfig: jupyter
+    spec:
+      # Prefer running pods on different nodes for redundancy
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 100
+              podAffinityTerm:
+                labelSelector:
+                  matchExpressions:
+                    - key: deploymentconfig
+                      operator: In
+                      values:
+                        - "jupyter"
+                topologyKey: kubernetes.io/hostname
+{% set image_pull_secret_name = jupyter_image_pull_secret_name | default(default_image_pull_secret_name) %}
+{% if image_pull_secret_name | length %}
+      imagePullSecrets:
+        - name: "{{ image_pull_secret_name }}"
+{% endif %}
+      containers:
+        - name: jupyter
+          image: "{{ jupyter_image_name }}:{{ jupyter_image_tag }}"
+          imagePullPolicy: Always
+          envFrom:
+            - secretRef:
+                name: "{{ jupyter_secret_name }}"
+          env:
+            - name: JUPYTER_ENABLE_LAB
+              value: "yes"
+          volumeMounts:
+            - name: jupyter-v-home
+              mountPath: /home/jovyan
```

Comment:
Can you add a comment explaining why this particular path ? I suppose this home dir is created in the image used but a comment will be better here IMO

Reply:
This home directory is required to store files created in working sessions (notebooks, data, etc.).